lib/propertiesPanel/JSONDataProperty.js

```diff
@@ -105,7 +106,10 @@ const JSONDataProperty = (props) => {
   return TextAreaEntry({
     element,
     label: translate('Example Output Data'),
-    description: translate('A map of variables provided to the engine upon task completion. If specified, it is used to provide variable suggestions in output mappings and/or the process. Must be a valid JSON object.'),
+    description: html`
+    <a href="https://docs.camunda.io/docs/next/components/modeler/data-handling/" target="_blank" rel="noopener" title=${ translate('Example Data documentation') }>
+      ${ translate('How to add example data') }
```

Comment:
The description only consisting of a link is in line with other documentation we have in the properties panel, e.g. Task Definition: 
![image](https://user-images.githubusercontent.com/21984219/233997525-b3a22656-2601-46f8-a2a9-3cdeb979801b.png)

Reply:
We could even simplify this to "About example data". What do you think?

Reply:
![image](https://user-images.githubusercontent.com/21984219/234001579-5fbeab91-f2e0-4985-bf01-1df7a814635a.png)

I like it 👍 

Added with [90e3fcc](https://github.com/camunda/example-data-properties-provider/pull/11/commits/90e3fccaa4c704e36e9c84c7f527a93063a9351b)

Reply:
Ahh and it should be `Example data` (lower case `d`).

Reply:
Indeed, it should be lower case to be consistent with other Section titles. I'll adjust it.

I am just wondering why we don't use [Title case](https://en.wikipedia.org/wiki/Title_case) in the Properties panel

Reply:
> I am just wondering why we don't use [Title case](https://en.wikipedia.org/wiki/Title_case) in the Properties panel

Me too. But consistency is more important here.